react/src/components/UserProfileSettingModal.tsx

```diff
@@ -277,37 +258,84 @@ const UserProfileSettingModal: React.FC = () => {
           <Form.Item
             name="newPasswordConfirm"
             label={t('webui.menu.NewPasswordAgain')}
+            dependencies={['newPassword']}
             rules={[
               ({ getFieldValue }) => ({
                 validator(_, value) {
                   if (!value || getFieldValue('newPassword') === value) {
                     return Promise.resolve();
                   }
                   return Promise.reject(
-                    new Error(t('environment.PasswordsDoNotMatch')),
+                    new Error(t('webui.menu.NewPasswordMismatch')),
                   );
                 },
               }),
             ]}
           >
             <Input.Password />
           </Form.Item>
-          {totpSupported ? (
-            <Form.Item label={t('webui.menu.TotpActivated')}>
+          {!!totpSupported && (
+            <Form.Item
+              name="totp_activated"
+              label={t('webui.menu.TotpActivated')}
+              valuePropName="checked"
+            >
               <Switch
-                defaultChecked={totpActivated}
-                onChange={(e) =>
-                  totpActivated
-                    ? dispatchEvent('confirmRemovingTotp', null)
-                    : dispatchEvent('startActivatingTotp', null)
-                }
+                onChange={(checked: boolean) => {
+                  if (checked) {
+                    toggleTOTPActivateModal();
+                  } else {
+                    if (user?.totp_activated) {
+                      form.setFieldValue('totp_activated', true);
+                      modal.confirm({
+                        title: t('totp.TurnOffTotp'),
+                        icon: <ExclamationCircleFilled />,
+                        content: t('totp.ConfirmTotpRemovalBody'),
+                        okText: t('button.Yes'),
+                        okType: 'danger',
+                        cancelText: t('button.No'),
+                        onOk() {
+                          mutationToRemoveTotp.mutate(userInfo.email || '', {
```

Comment:
This occurs compiled error @yomybaby 
![image](https://github.com/lablup/backend.ai-webui/assets/28584164/e830fab5-5ae6-46a7-b3c5-5ae039f7edba)

Reply:
It's already resolved. Sorry for forgetting to push.